docs/help_classes.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Help Classes
+---
+Out of the box oclif provides a great help experience provided by @oclif/plugin-help. This can be customized or replaced with custom help classes. To get started make sure at least @oclif/command (*REPLACE WITH PUBLISHED VERSION*) and version 3 of @oclif/plugin-help are installed.
+
+```
+yarn add --latest @oclif/plugin-help@3
+yarn add --latest @oclif/command
+```
+
+Note: Previously, in version 2 of @oclif/plugin-help topic and commands would be combined under one *COMMANDS* list for root, topic and command help output. In version 3 of @oclif/plugin-help the new default `Help` class splits these child topics and commands into distinct *TOPICS* and *COMMANDS* lists. The *TOPICS* list includes any direct children that contains their own child topics or commands, while the *COMMANDS* list represents direct children that are runnable commands.
+
+Note: These examples will follow a typescript project but you can find a [list of the differences](#setting-up-extended-help-classes-in-javascript-projects) below that would apply to a javascript project.
+
+To get started a path to the file containing the help class must be defined. This is a relative path to the help class, without an extension, in the "oclif" section of your package.json.
+
+For this example the help class will be created in a file at "[project root]/src/help.ts" and the package.json would be updated to include the path:
+```json
+{
+  ...
+  "oclif": {
+    "helpClass": "./lib/help"
+  }
+}
```

Comment:
I think we'd be better off without the `...` given how much unnecessary attention it demands with the syntax error. Or if you want to illustrate that there are other options, maybe...

```js
{
  arbitrary,
  other,
  properties,

  "oclif" : {
    "helpClass": "./lib/help"
  }
```

or even simply

```js
{
  //...
  "oclif" : {
    "helpClass": "./lib/help"
  }
```